### PR TITLE
Fixes bugs for SetDefaultShiftTimings and SetRoleReq Commands

### LINKED ISSUE #151: Improve Code Quality for implementation of Role Requirements
Remove hard code in ViewShift output, RoleReqStorage etc where the order of roles is assumed to be: Bartenders, Floor, Kitchen

### LINKED ISSUE #160: [PE-D] Inconsistency in UG
Command and the one used in the example are different

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: whoisjustinngo/ped#2

### LINKED ISSUE #167: [PE-D] Typo in description
Should be just "staff" instead

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: whoisjustinngo/ped#9

### LINKED ISSUE #172: [PE-D] setShiftTimings command does not exist?
Steps to reproduce:

1. Type in `setShiftTimings 10:00 16:00 17:00 22:00`

Expected Result: Shift timing should be set

Actual Result: Unknown Command

-------------
Labels: `type.FunctionalityBug` `severity.High`
original: RachelCheah/ped#2

### LINKED ISSUE #190: [PE-D] setRoleRequirement error message not updated properly
before this screenshot the floor was set to 8, however this error message still displays 0

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: yulonglim/ped#3

### LINKED ISSUE #191: [PE-D] Setting default shift timings
I followed the exact command on the UG but I got the Unknown Error command. 
I tried with the command `SetDefaultShiftTimings` so I believe its just an update needed on the UG!

-------------
Labels: `type.FunctionalityBug` `severity.High`
original: jason-ng-zq99/ped#1

### LINKED ISSUE #192: [PE-D] UG example are wrong ( setDefaultShiftTiming vs setShiftTiming)
-------------
Labels: `severity.High` `type.DocumentationBug`
original: yulonglim/ped#4

### LINKED ISSUE #214: [PE-D] SetRoleRequirements is not reset after clearing addressbook
-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: yulonglim/ped#10

### PR DESCRIPTION
**Pull request by mweeruien**
Current Fixes:
- Fix UG Examples (previously said "setShiftTimings")
- Fix bug where Help Message for SetDefaultShiftTimings Command was not updated properly
- Resets the ShiftTimings and RoleReq to the default when the Clear Command is called
- Fix typo in description of viewSchedule in the UG (staff(s) instead of staff was used)
- Improve code quality for SetRoleReq and its storage so that it has fewer "magic numbers"

[UG](https://mweeruien.github.io/tp/UserGuide.html) For reference.

Closes #151
Closes #160
Closes #167
Closes #172
Closes #190 
Closes #191
Closes #192
Closes #214

### CODE REVIEW

**Review by IrvingHe000 — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -366,7 +385,7 @@ Examples:
 `change da/2021-12-28`  
 
 
-#### Viewing schedule of staff(s): `viewSchedule`
+#### Viewing schedule of staff: `viewSchedule`
```
Comment (Gabau) [inline docs/UserGuide.md:388]: I don't really understand the problem with this? viewSchedule command can view the schedule of multiple staff?

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM. I do not really understand #167. Shouldn't the two staff be staff(s) as we can view for multiple or singular?

File: docs/UserGuide.md (hunk 1 above)
Comment (mweeruien) [reply docs/UserGuide.md:388]: It was just a UG bug someone reported, but I can revert if if ya want?

Comment (Gabau) [reply docs/UserGuide.md:388]: I think the typo in #167 is referring to `view the schedule of staff's ...` should be `view the schedule of staffs ...`

Comment (mweeruien) [reply docs/UserGuide.md:388]: Ohh yea that makes sense, thank you!

**Review by Gabau — APPROVED** [review]
LGTM

**Pull request merged**